cli.js

```diff
@@ -1,12 +1,47 @@
 #!/usr/bin/env node
-import fs from 'node:fs'
 import { globbySync } from 'globby'
+import fs from 'node:fs'
 import sortPackageJson from './index.js'
 
 const isCheckFlag = (argument) => argument === '--check' || argument === '-c'
+const isHelpFlag = (argument) => argument === '--help' || argument === '-h'
+const isVersionFlag = (argument) =>
+  argument === '--version' || argument === '-V'
 
 const cliArguments = process.argv.slice(2)
 const isCheck = cliArguments.some(isCheckFlag)
+const isHelp = cliArguments.some(isHelpFlag)
+const isVersion = cliArguments.some(isVersionFlag)
+
+if (isHelp) {
+  console.log(
+    `Usage: sort-package-json [OPTION...] [FILE...]
+Sort npm package.json files. Default: ./package.json
+Strings passed as files are parsed as globs.
+
+  -c, --check                check if FILES are sorted
+  -q, --quiet                don't output success messages
+  -h, --help                 display this help and exit
+  -V, --version              display the version and exit
+  `,
+  )
+  process.exit(0)
+}
+
+if (isVersion) {
+  try {
+    const cliRealPath = fs.realpathSync(process.argv[1])
+    const cliParentDir = (await import('node:path')).dirname(cliRealPath)
+    const packageJsonBuffer = fs.readFileSync(`${cliParentDir}/package.json`)
```

Comment:
```suggestion
    const packageJsonBuffer = fs.readFileSync('./package.json')
```

Reply:
This suggestion won't work because the CLI can be run from any location, but I need to read the sort-package-json/package.json file specifically. Replacing it with the current directory (.) will result in either an error (if no package.json file is present) or reading another package's version number.

Reply:
I've been struggling with that very line for a while, and this is the best solution I could come up with.

Reply:
Having done some more research, I have greatly simplified the process of finding the project.json file using import.meta.url.

Reply:
Sorry, you are right. I don't know what am I thinking. Using `import.meta.url` is the right way.